warehouse/packaging/utils.py

```diff
@@ -107,12 +107,12 @@ def render_simple_detail(project, request, store=False):
         f"{project.normalized_name}/{content_hash}.{project.normalized_name}.html"
     )
 
-    if store:
-        storage = request.find_service(ISimpleStorage)
-        with tempfile.NamedTemporaryFile() as f:
-            f.write(content.encode("utf-8"))
-            f.flush()
+    with tempfile.NamedTemporaryFile() as f:
+        simple_detail_size = f.write(content.encode("utf-8"))
+        f.flush()
```

Comment:
Is this the only way for us to measure the size by writing out to disk? Could we keep the writing to disk still behind the `if store` conditional, and calculate the size in memory?

Reply:
Addressed in `refactor: utils.render_simple_detail`